doc/source/installation.rst

```diff
@@ -10,9 +10,12 @@ Use::
 
 Pip will install :
 
-1. The library and all dependencies into your site-packages directory
-2. Redfish client master conf file into /etc/redfish_client.conf
-3. Data file (templates) into /usr/share/redfish-client/templates
+1. The library and all dependencies into prefix/lib/pythonX.Y/site-packages directory
+2. Redfish client master conf file into prefix/etc/redfish_client.conf
+   Unless if prefix = '/usr', in that case force configuration file to be in /etc except
+   for Debian based system
```

Comment:
Debian is using specific destination for their packages : /usr/local/lib/pythonX.Y/dist-packages
However the prefix is still /usr
Currently that's the issue I have. I though that distutils.sysconfig.PREFIX will give me the correct installation prefix. But this is not always the case.... Unfortunately I noticed that a bit too late. :(
So still working to fine a reliable way to define the target location.